pytket/pytket/circuit/display/__init__.py

```diff
@@ -66,10 +69,34 @@ def render_circuit_as_html(
 
 
 def render_circuit_jupyter(
-    circuit: Union[Dict[str, Union[str, float, dict]], Circuit],
+    circuit: RenderCircuit,
 ) -> None:
     """Render a circuit as jupyter cell output.
 
     :param circuit: the circuit to render.
     """
     render_circuit_as_html(circuit, True)
+
+
+def view_browser(circuit: RenderCircuit, browser_new: int = 2, sleep: int = 5) -> None:
+    """Write circuit render html to a tempfile and open in browser.
+
+    Waits for some time for browser to load then deletes tempfile.
+
+    :param circuit: the circuit to render.
```

Comment:
Here we should describe the type (either a `Circuit` or (I assume) a serialized `Circuit`).